demo-webon/test/web3_signing_test.ts

```diff
@@ -23,7 +23,7 @@ export async function testSigning(): Promise<string> {
     return "This test only works with the reject wallet";
   }
 
-  const res = await nomo.signEvmTransaction({ messageHex: inputHex });
+  const res = await nomo.signEvmTransaction({ messageHex: inputHex, chainID: "383414847825" });
```

Comment:
since we already have passing TXs on both Ethereum-mainnet and ZENIQ Smarchain, it seems mathematically impossible that the interface is "lacking information".
So I doubt that this interface expansion is really needed, also it would break lots of existing WebOn if that was going to be mandatory.